src/perl/perl-core.c

```diff
@@ -354,10 +357,11 @@ PERL_SCRIPT_REC *perl_script_find_package(const char *package)
 /* Returns full path for the script */
 char *perl_script_get_path(const char *name)
 {
-	struct stat statbuf;
 	char *file, *path;
 
-	if (g_path_is_absolute(name) || (name[0] == '~' && name[1] == '/')) {
+	if (g_path_is_absolute(name) ||
+	    (name[0] == '~' && name[1] == '/') ||
+	    (name[0] == '.' && name[1] == G_DIR_SEPARATOR)) {
```

Comment:
Apparently all the versions of windows derived from NT support / as a path separator. So, everything newer than 9x/ME. I don't think we support non-cygwin windows anyway.

[x] burninate